central/scannerdefinitions/handler/handler.go

```diff
@@ -67,6 +69,8 @@ var (
 		"name2cpe": "repomapping-tmp/container-name-repos-map.json",
 		"repo2cpe": "repomapping-tmp/repository-to-cpe.json",
 	}
+
+	v4VersionRegex = regexp.MustCompile(`^(?:\d+\.\d+\.(?:\d+|x(?:-\d+-g[0-9a-f]+|-nightly-\d{8}))|dev)$`)
```

Comment:
instead of a regexp, you can probably just hardcode a key, right?

Reply:
I don't quite understand how to do that. I want something just like the UUID, each uuid and v4 version can have one associated updater.